$OpenBSD: patch-plugins_zeitgeist_zeitgeist_plugin_desktop_in,v 1.1 2014/03/27 12:17:43 jasper Exp $

Unbreak loading the zeitgeist plugin:

(gedit:15316): libpeas-WARNING **: zeitgeistplugin: File not found

(gedit:15316): libpeas-WARNING **: Error loading plugin 'zeitgeistplugin'

--- plugins/zeitgeist/zeitgeist.plugin.desktop.in.orig	Thu Mar 27 13:12:29 2014
+++ plugins/zeitgeist/zeitgeist.plugin.desktop.in	Thu Mar 27 13:12:33 2014
@@ -1,5 +1,5 @@
 [Plugin]
-Module=zeitgeistplugin
+Module=zeitgeist
 IAge=3
 _Name=Zeitgeist dataprovider
 _Description=Logs access and leave event for documents used with gedit
